elide-async/src/test/java/com/yahoo/elide/async/service/AsyncQueryCleanerThreadTest.java

```diff
@@ -58,15 +59,19 @@ public void testAsyncQueryCleanerThreadSet() {
 
     @Test
     public void testDeleteAsyncQuery() {
+        ArgumentCaptor<FilterExpression> filterCaptor = ArgumentCaptor.forClass(FilterExpression.class);
         cleanerThread.deleteAsyncQuery();
-
-        verify(asyncQueryDao, times(1)).deleteAsyncQueryAndResultCollection(anyString());
+        verify(asyncQueryDao, times(1)).deleteAsyncQueryAndResultCollection(filterCaptor.capture());
+        Long date = System.currentTimeMillis() - (cleanerThread.getQueryCleanupDays() * 24 * 3600 * 1000);
```

Comment:
It's possible for this test to fail because of a 1ms gap between when this date is calculated and when the delete filter is generated.   We'll need to find a less brittle approach.